glTFLoader/Interface.cs

```diff
@@ -337,7 +337,8 @@ public static void SaveModel(this Gltf model, Stream stream)
         {
             string fileData = SerializeModel(model);
 
-            using (var ts = new StreamWriter(stream, leaveOpen: true))
+            // https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/IO/StreamWriter.cs,93
+            using (var ts = new StreamWriter(stream, null, -1, leaveOpen: true))

```

Comment:
```
3>.\KhronosGroup\glTF-CSharp-Loader\glTFLoader\Interface.cs(341,33,341,45): error CS7036: There is no argument given that corresponds to the required formal parameter 'encoding' of 'StreamWriter.StreamWriter(Stream, Encoding, int, bool)'
Project glTFLoader (netstandard1.3)
```

Reply:
I don't know exactly what is going on, but the source code link you referenced should work with this syntax, but the metadata doesn't have default parameters which is why it doesn't compile. I'm not sure why there is a discrepancy between the metadata and the source code. Either way, the comment isn't helping, so I would recommend removing it.

```suggestion
            using (var ts = new StreamWriter(stream, null, -1, true))
```

Reply:
This isn't right either. The reason why the previous syntax doesn't work is because the target framework version doesn't have this constructor overload. I'm trying a few things.

Reply:
I created a [PR](https://github.com/ericwj/glTF-CSharp-Loader/pull/1) to your branch to fix this. We either have to update the framework versions or use an older syntax.

Reply:
The proper solution would be to file a bug for this, but that could take a lot of effort and waiting for very little benefit. I think lets make it build rather than bumping the TFM. Something new and better could later justify shutting out .NET Framework.

Reply:
```cs
public static void SaveModel(this Gltf model, Stream stream)
{
    string fileData = SerializeModel(model);

    const Encoding DefaultEncoding = null; // cached UTF8NoBOM
    const int DefaultBufferSize = 1024;
    using (var ts = new StreamWriter(stream, DefaultEncoding, DefaultBufferSize, leaveOpen: true))
    {
        ts.Write(fileData);
    }
}
```

Reply:
I get this error with these changes:
```
1>------ Build started: Project: glTFLoader, Configuration: Debug Any CPU ------
1>glTFLoader -> D:\GitHub\glTF-CSharp-Loader\glTFLoader\bin\Debug\netstandard1.3\glTFLoader.dll
1>D:\GitHub\glTF-CSharp-Loader\glTFLoader\Interface.cs(342,90,342,99): error CS1739: The best overload for 'StreamWriter' does not have a parameter named 'leaveOpen'
```

Reply:
Weird, it builds on my machine with both SDK's.
```json
{
  "sdk": {
    "version6": "6.0.400",
    "version7": "7.0.100-preview.7.22377.5",
    "version": "6.0.400"
  }
}
```
`global.json`

Reply:
The tooltip says this:
<img width="334" alt="image" src="https://user-images.githubusercontent.com/19158740/187472367-6b9d405c-59db-4aad-8342-31909ba2017c.png">

Reply:
Yes I removed `net35`. It's dead.

Like you said `.nuget\packages\system.io\4.3.0\ref\netstandard1.3\System.IO.dll` is missing the defaults - the arguments for the constructor aren't optional.

But what is the big deal here? It'll work. It'll be the same. Lets move on.

Reply:
I didn't pull your latest changes yet, so I still had `net35` in the list. Are we going to break users if we remove .NET framework support?

> But what is the big deal here? It'll work. It'll be the same. Lets move on.

I'm just trying to build your PR locally.

Reply:
Also, if we are going to target just one framework. We should change it to `<TargetFramework>netstandard1.3</TargetFramework>` (i.e. without the `s`) so that VS will show it better in the project properties window.

Reply:
> Are we going to break users if we remove .NET framework support?

I have zero knowledge of who is using this. I can only see 3800 downloads from NuGet and that the last update was 2015 so its really small and for good reason.

Well one of those is me and I got a `TypeLoadException`. Anyone getting that is just tossing the whole thing I reckon.

> I'm just trying to build your PR locally.

If you have the SDK version I listed here it should build fine. Or something really weird is going. If you try to build for `net35` still then just stop doing that lol. I commented that TFM out as about the first thing I did.

Reply:
OK, I ran this latest commit against .NET Core 1.0.16, .NET Framework 4.6 and .NET 6.0. It works.

NUnit crashes without error on old .NET Framework, so I ported to Xunit. The tests are still limited to supported versions of .NET though, `netcoreapp3.1`, `net462` and `net6.0`.

Reply:
I don't know if you need to touch the unit tests. They are not part of the public contract, so it just need to work when running. I pulled your latest changes and I get this error for the unit tests:

> C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(1220,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.6.2 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks

I added it the targeting pack with the VS installer and then it worked. Since this isn't installed by default with VS, I'm not we should include it.

Looks good otherwise. Sorry for the long tail on this one. Thanks for all the work!

Reply:
It'll be the same for .NET Core 3.1 except for two ducks and a horsehead who still work with it. Seeing the acquisition is apparently quite easy, I'd leave it like this. Just get it out there so people don't get `TypeLoadException` anymore.